package.json

```diff
@@ -53,7 +65,13 @@
     "prosemirror-view": "^0.20.4",
     "react-syntax-highlighter": "^5.5.0",
     "semantic-release": "^6.3.2",
-    "webpack": "^2.4.1"
+    "webpack": "^2.4.1",
+    "babel-jest":"^19.0.0",
```

Comment:
https://github.com/facebook/jest/tree/master/packages/babel-jest
Isn't that supposed to have transform add to jest config:
```js
"transform": {
  "^.+\\.jsx?$": "babel-jest"
},
```